platform/node/README.md

```diff
@@ -36,9 +36,9 @@ var mbgl = require('@mapbox/mapbox-gl-native');
 var sharp = require('sharp');
 
 var options = {
-  request: function(req, callback) {
+  request: function(req) {
     fs.readFile(path.join(__dirname, 'test', req.url), function(err, data) {
-      callback(err, { data: data });
+      req.respond(err, { data });
```

Comment:
Can you explain a little bit more on the nature of this change?  This is something that will impact downstream users, but the nature of `req` in this context isn't immediately obvious - compared to a simple callback.

Reply:
Yeah, so this was actually not necessary - I’ll revert this back 👍